app/models/comfy/cms/page.rb

```diff
@@ -37,9 +44,16 @@ def related_links(limit)
     Comfy::Cms::Page
       .unscoped
       .all_english_articles
+      .ignore_suppressed
       .sort_by_tag_similarity(keywords)
       .remove_self_from_results(self)
       .where.not(id: mirrored_page.try(:id))
       .limit(limit)
   end
+
+  def suppress_mirrors_from_links_recirculation
+    mirrors.each do |m|
+      m.update_column(:suppress_from_links_recirculation, suppress_from_links_recirculation)
+    end
+  end
```

Comment:
Btw, is the comfy not mirroring automatically the suppress_from_links_recirculation? Do we need this method?